docs/docs/APIGuide/Layers/Recurrent-Layers.md

```diff
@@ -1558,4 +1561,232 @@ array([[[[ 0.1       ,  0.2       ],
         [[ 0.30000001,  0.40000001],
          [ 0.30000001,  0.40000001]]]], dtype=float32)
 ```
+---
+
+## MultiRNNCell ##
+
+**Scala:**
+```scala
+val model = MultiRNNCell(cells = multiRNNCells)
```

Comment:
type of cells?

Reply:
Updated

Reply:
You need to specify that cells should be an array of Cell